src/components/task-item/template.html

```diff
@@ -0,0 +1,34 @@
+<link
+  rel="stylesheet"
+  href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
```

Comment:
just gonna call this out again to keep it in our minds about using a local ref for this

Reply:
Ok. I will research a way to download this during build time.